app/js/components/EscrowList.jsx

```diff
@@ -0,0 +1,92 @@
+/*global web3*/
+import React, {Fragment} from 'react';
+import {Card, CardBody, CardHeader, CardTitle, Table, Button, Alert} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+const escrowStates = {
+  released: 'released',
+  canceled: 'canceled',
+  expired: 'expired',
+  waiting: 'waiting'
+};
+
+function getEscrowState(escrow) {
+  if (escrow.released) {
+    return escrowStates.released;
+  }
+  if (escrow.canceled) {
+    return escrowStates.canceled;
+  }
+  if (escrow.expired <= Date.now() / 1000) {
+    return escrowStates.expired;
+  }
+  return escrowStates.waiting;
+}
```

Comment:
If we ever show an escrow that could be usefull to be able to reach those function, we could place them for example in:
features/escrow/helper.js

What do you think?

Reply:
If I understand correctly, you're asking to move those to another file case other escrow files need them? 
If so, yeah it makes sense